src/sentry/static/sentry/app/components/events/errors.tsx

```diff
@@ -190,7 +195,7 @@ class EventErrors extends React.Component<Props, State> {
             <EventErrorItem key={errorIdx} error={error} />
           ))}
 
-          {project?.features?.includes('reprocessing-v2') && (
+          {'features' in project && project?.features?.includes('reprocessing-v2') && (
```

Comment:
```suggestion
          {project?.features?.includes('reprocessing-v2') && (
```

Reply:
I can't do this, typescript complains.

Reply:
this is because the type `features` is not present in projects. Please see:

![image](https://user-images.githubusercontent.com/29228205/93082884-fb001980-f691-11ea-9bab-173745bd6ec1.png)

As you are touching this file, maybe you could add it?

Reply:
The type really does not exist on AvatarProject and is also not supposed to exist there

Reply:
you can do as below:

`  {(project as Project)?.features?.includes('reprocessing-v2') && (`